src/App.tsx

```diff
@@ -13,8 +13,20 @@ export class App extends React.Component<{}, State> {
     movies: moviesFromServer,
   };
 
-  addMovie = (/* movie: Movie */) => {
-    // put your code here
+  addMov = (newMov: any) => {
+    if (newMov) {
+      this.setState(state => {
+        return ({
+          movies: [...state.movies, newMov],
+        });
+      });
+    } else {
+      this.setState(state => {
+        return ({
+          movies: [...state.movies],
+        });
+      });
```

Comment:
```suggestion
  addMov = (newMov: any) => (
      this.setState(state => ({
          movies: [...state.movies, newMov],
        });
     );
```
And don't use type 'any'